packages/@okta/i18n/src/properties/login.properties

```diff
@@ -915,6 +915,9 @@ oie.phone.invalid = Invalid Phone Number.
 oie.email.label = Email
 oie.email.authenticator.description = Verify with a link or code sent to your email
 oie.email.mfa.title = Verify with your email
+oie.email.verify.primaryButton = Send me an email
```

Comment:
Just want to callout that we should make sure the translations and the code changes go out in the same release as I am assuming these are customer visible changes for OIE that are scheduled to go out as part of the monthly major  release. 

We usually get translations a [week before the monthly release](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/741293778/Requesting+and+Receiving+Translations+i18n+and+l10n#When-would-I-get-new-translations%3F) so these new strings should get included.

Reply:
This specific change is driven by the API and the API changes are behind a flip. The FF would be turned on in December.